charts/simple-app/values.yaml

```diff
@@ -51,7 +55,7 @@ proxySidecar:
     tag: latest
 
     # -- (String) Always, Never or IfNotPresent
-    pullPolicy: Always
+    pullPolicy: IfNotPresent
```

Comment:
does `IfNotPresent` cause problems with a deploy?  hopefully we're using uniquely tagged containers for deployments, but if someone was using `:latest` this would not necessarily get updates.

Reply:
This setting is for the optional `proxySidecar` - the main application image is already set to use `IfNotPresent`. These are overridable of course - but in general I find the `Always` pull practice to be bad because it just slows down the deploy process.